students/JasneetChandok/Lesson05/mailroom3.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3.6
+
+'''Lesson 5 - Mailroom Part 3'''
+
+# Importing Systems Module
+try:
+    import sys
+except ImportError:
+    print("The sys module could nout be imported!")
+
+
+# Starter list of existing donors
+donors = {'jasneet paramjit': [50, 200, 500],
+          'simran kaur': [100, 200, 500],
+          'manikaran chandok': [100, 200, 500, 680],
+          'manveen chandok': [10, 200, 65],
+          'sanjam patwalia': [200, 50]}
+
+
+# List of existing donor
+def existingdonors_list():
+    donor_list = "\n ".join(donor.title()for donor in donors)
+    print("\nList of Existing Donors:", "\n", donor_list, "\n")
+
+
+# Add donation to an existing donor
+def add_donation():
+    try:
+        donor_selection = input("Enter 'Full Name' of the donor >")
+        donor_donation = input("\nEnter the 'Amount' donated >")
+        donor = donor_selection.strip().lower()
+        if donor in donors:
+            donor_amount = donors.get(donor)
+            donor_amount.append(int(donor_donation))

```

Comment:
You will want to be careful here. You can run into an exception if the donor_donation is a string!